core/src/modules/ops/destination.ts

```diff
@@ -926,23 +926,40 @@ export namespace DestinationOps {
     const locks: Awaited<ReturnType<typeof getLock>>[] = [];
 
     try {
+      const exportIds = await Export.sequelize
```

Comment:
It's pedantic, but this SQL is kind of complex - a better variable name might be something like `newestExportIds`